Demo/src/main/java/com/paypal/android/api/services/SDKSampleServerAPI.kt

```diff
@@ -234,15 +236,7 @@ class SDKSampleServerAPI {
         merchantIntegration: MerchantIntegration = SELECTED_MERCHANT_INTEGRATION
     ) = safeApiCall {
         val response = findService(merchantIntegration).createSetupToken(jsonObject)
-
-        val responseJSON = JSONObject(response.string())
-        val customerJSON = responseJSON.getJSONObject("customer")
-
-        PayPalSetupToken(
-            id = responseJSON.getString("id"),
-            customerId = customerJSON.getString("id"),
-            status = responseJSON.getString("status")
-        )
+        Gson().fromJson(response.string(), PayPalSetupToken::class.java)
```

Comment:
Can we revert `Gson` parsing in this PR and create a ticket to fully adopt `Gson` in the demo app?